### ISSUE TITLE
Clearer description for "Set Parent to Anchor"

### ISSUE THREAD

**Issue by aggixx** [WeakAuras/WeakAuras2#656]
**Is your feature request related to a problem? Please describe.**
It took me a good 30 minutes or so to figure out why a relative simply WA wasn't showing when I expected it to, and it turns it was because "Set Parent to Anchor" was enabled and I hadn't really thought about what that actually did.

And this is coming from someone who has done quite a fair amount of work with addons, so I've dealt with frames a bit and get what parenting does. However, the average weakaura user surely has no idea what parenting does and therefore what this option does to them would be a mystery.

**Describe the solution you'd like**
A tooltip description for the option, I suggest something along these lines

> Sets the anchored frame as the aura's parent, causing the aura to automatically inherit attributes such as visiblility, etc.

Not sure what the exhaustive list of things parent-child relationship affects but I imagine visibility is the biggest one and you could tack a few more onto that list on the end that might be a typical use scenario for this option.

**Describe alternatives you've considered**
Maybe reword the option text itself, although its already quite concise and accurate.

**Additional context**
https://discordapp.com/channels/172440238717665280/172440238717665280/480787992676663316

**Comment by emptyrivers** [comment]
The next release will have a tooltip description of "Sets the anchored frame as the aura's parent, causing the aura to inherit attributes such as visibility and scale."
